.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
```

Comment:
Is that really necessary

Reply:
Yes, sometimes you want to clarify why you can't do it like X. As I said above, if you don't need it, delete it.

Reply:
Okay, but please change the heading to "Considered Alternatives".

Reply:
-> 97de6f839a26c4953b901b7a1db368516eff21c6